lib/mpl_toolkits/tests/test_axes_grid.py

```diff
@@ -12,10 +12,16 @@
 # The original version of this test relied on mpl_toolkits's slightly different
 # colorbar implementation; moving to matplotlib's own colorbar implementation
 # caused the small image comparison error.
-@pytest.mark.parametrize("legacy_colorbar", [False, True])
-@image_comparison(['imagegrid_cbar_mode.png'],
+@image_comparison(['imagegrid_cbar_mode.png'] * 2,
```

Comment:
```suggestion
@image_comparison(['imagegrid_cbar_mode.png',
                   'imagegrid_cbar_mode.png'],
```
Slight preference for writing out the duplication to make it more obvious. But won‘t block if you think compact is better.

Reply:
I was just lazy and trying to see if the Fedora package would work quickly. Writing out twice is fine.